mlir/include/mlir/Dialect/EmitC/IR/EmitCTypes.td

```diff
@@ -130,4 +130,16 @@ def EmitC_PointerType : EmitC_Type<"Pointer", "ptr"> {
   let assemblyFormat = "`<` qualified($pointee) `>`";
 }
 
+class EmitC_SizeType<string name, string typeMnemonic, list<Trait> traits = []> 
+  : EmitC_Type<name, typeMnemonic, traits> {
+}
+
+def EmitC_SignedSizeType : EmitC_SizeType<"SignedSize", "ssize_t"> {
+  let summary = "EmitC signed size type";
+}
+
+def EmitC_UnsignedSizeType : EmitC_SizeType<"UnsignedSize", "usize_t"> {
```

Comment:
`usize_t` is pretty uncommon; can we use `size_t` to be less confusing to people that know `size_t`?

Reply:
This "u" makes its unsigned nature stand out. Would it be OK to have a "size_t" mnemonic and the class named "UnsignedSizeType"?